piwheels/initdb/sql/create_piwheels.sql

```diff
@@ -1082,6 +1183,29 @@ $sql$;
 REVOKE ALL ON FUNCTION test_package_version(TEXT, TEXT) FROM PUBLIC;
 GRANT EXECUTE ON FUNCTION test_package_version(TEXT, TEXT) TO {username};
 
+-- get_package_versions_deleted(package)
```

Comment:
`get_versions_deleted maybe`? Just feels a bit long, but I'd be happy to leave this as is.